harness/config/commands.yml

```diff
@@ -112,6 +112,15 @@ command('console'):
     #!bash(workspace:/)|@
     passthru $COMPOSE_BIN exec -u "${CODE_OWNER}" console bash
 
+command('root <service>', 'root'):
+  description: Open a default root shell on any running container
+  env:
+    COMPOSE_BIN: = @('docker.compose.bin')
+  exec: |
+    #!bash(workspace:/)|=
+    loginshell=$(docker-compose -p ={ @('namespace') } exec -T ={ input.argument('service') } cat /etc/passwd | grep '^root:' | sed -E 's/^.*:([^:]+)$/\1/')
```

Comment:
One more docker-compose direct usage here.

Can set env var `COMPOSE_PROJECT_NAME: = @('namespace')` to avoid the `-p` call here too, unless somehow that's taken care of in the docker harness now.

Reply:
Oh that's an oversight, I'll fix that. `COMPOSE_PROJECT_NAME` has been superseded though https://github.com/inviqa/harness-docker/commit/403c6ee67ac95c5f4a323bd539abde5d8d170320